resources/js/components/Product/AddToCart.vue

```diff
@@ -3,6 +3,21 @@ import { GraphQLError } from '../../fetch'
 import { mask, refreshMask } from '../../stores/useMask'
 
 export default {
+    inject: {
+        instantSearchInstance: {
+            from: '$_ais_instantSearchInstance',
+            default: () => {
+                return {
+                    helper: {
+                        state: {
+                            disjunctiveFacetsRefinements: {},
```

Comment:
This is the default value for when the addToCart is not within context of an `instantSearchInstance`